view/Cargo.toml

```diff
@@ -54,6 +54,7 @@ async-trait = "0.1"
 tendermint = "0.26.0"
 tendermint-rpc = { version = "0.26.0", features = ["http-client"] }
 sha2 = "0.10.1"
+url = "*"
```

Comment:
We probably want to specify a version here, right?

Reply:
Also done, pinning at `"2"`, so we'll continue to get minor and patch versions when updating.